crates/sp-consensus-subspace/src/lib.rs

```diff
@@ -304,7 +304,7 @@ sp_api::decl_runtime_apis! {
         fn is_in_block_list(farmer_public_key: &FarmerPublicKey) -> bool;
 
         /// Get MerkleRoot for specified segment index
-        fn merkle_tree_for_segment_index(segment_index: u64) -> Option<Sha256Hash>;
+        fn records_root_for_segment_index(segment_index: u64) -> Option<Sha256Hash>;
```

Comment:
How about this? @nazar-pc 

```suggestion
        fn records_root_for(segment_index: u64) -> Option<Sha256Hash>;
```

Reply:
I think we can even simplify it to this since segment index is the only way to address records root anyway:
```suggestion
        fn records_root(segment_index: u64) -> Option<Sha256Hash>;
```